pytorch_toolkit/instance_segmentation/README.md

```diff
@@ -0,0 +1,263 @@
+# Instance Segmentation in PyTorch
+
+This repository contains inference and training code for Mask R-CNN like
+networks. Models code is designed to enable ONNX export (with custom operations)
+and inference on CPU via OpenVINO.
+[Detectron](https://github.com/facebookresearch/Detectron) and
+[maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark)
+models are wrapped to export their weights to ONNX and OpenVINO.
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu 16.04
+* Python 3.5.2
+* PyTorch 0.4.1
+* OpenVINO 2019 R1 with Python API
+
+### Installation
+
+To install required dependencies run
+
+```bash
+$ cat requirements.txt | xargs -n 1 -L 1 pip3 install
```

Comment:
Why dont use  'pip3 install -r requirements.txt'? 
And add install `requirements_onnx.txt` from model_optimizer directory.

Reply:
@AlexanderDokuchaev `pip3 install -r requirements.txt` does not work because some modules are dependencies for other ones and will be installed only at the end when all modules have been proceeded. Our command installs all modules from `requirements.txt` one by one. `ONNX` is not required module, you can use the repo without this one (without conversion to `inference engine` format). If you need to use `inference engine` you need to install `OpenVINO 2019 R1 with Python API` (look `Prerequisites`) with all dependencies and modules including `model_optimizer`.